src/main/scala/io/viash/config/Config.scala

```diff
@@ -365,6 +365,16 @@ object Config extends Logging {
 
     val allConfigs = allConfigsWithStdOutErr.collect({ case Left((config, _, _)) if config.functionality.isEnabled => config })
 
+    // Verify that all configs except the disabled ones are unique
+    // Even configs disabled by the query should be unique as they might be picked up as a dependency
+    // Only allowed exception are configs where the status is set to disabled
+    val uniqueConfigs = allConfigs.groupBy(config => config.functionality.namespace.map(_ + "/").getOrElse("") + config.functionality.name)
```

Comment:
Perhaps you could also add organisation to this id. Wdyt?

Reply:
But the organisation isn't used in the output path, so collisions could still happen then.
However, this reminds me that I wanted to look into using `ViashNamespace.targetOutputPath(...)` as the key generator - as that is what determines what would result in collisions and means that once we have more flexibility in the folder name strategy as previously discussed, we're halfway there in solving that too.

Reply:
I see -- good point.

Should we even consider the use-case where one might use multiple organisations in one project -- perhaps not?

Reply:
The one I see most likely to happen is a project that sets the organisation but with one or more components where the organisation is not set. In that regard it's still 2 different values for the organisation.
In itself it doesn't really hurt much but it's probably an error.

If we want a check for that, I'd suggest displaying a warning and adding it to Viash 0.9 (or later).